ISISScientificComputing/autoreduce
# Refactor stomp and icat clients

**Description of changes**
* The stomp_client and the icat_client have been refactored and added to the utils package for use by other classes

**How to test**
* Pull the code onto the dev node and ensure all still functions correctly (note you will need to create a settings.py file `utils/clients/settings.py`)

Fixes #36

-------


**To test**
* Log into the devolpement nodes
* `git checkout this-pull-request-branch`
* restart services on each node (information on how to do this can be found in the development documentation)
* Submit some runs with the manualsubmission.py script

Once you are happy, merge this pull request into develop, and update the development nodes to the devlop branch
```
> git checkout origin/devlop
> git fetch -p
> git pull origin/develop
```
